firefox-ios/Client/Coordinators/Browser/BrowserCoordinator.swift

```diff
@@ -134,6 +134,7 @@ class BrowserCoordinator: BaseCoordinator,
         self.legacyHomepageViewController = legacyHomepageViewController
         legacyHomepageViewController.scrollToTop()
         // We currently don't support full page screenshot of the homepage
```

Comment:
in theory this comment should be moved down since it is about the `screenshotService`

Reply:
Yes true i'm going to move it down 😃